lib/event_tracer/appsignal_logger.rb

```diff
@@ -7,6 +7,7 @@
 #
 # Usage: EventTracer.register :appsignal, EventTracer::AppsignalLogger.new(Appsignal)
 #        appsignal_logger.info appsignal: { increment_counter: { counter_1: 1, counter_2: 2 }, set_gauge: { gauge_1: 1 } }
+#        appsignal_logger.info appsignal: { set_gauge: { gauge_1: { value: 1, region: 'eu' } } }
```

Comment:
just a minor note, we use `attribute.fetch(:tags)` to fetch the tag value but in this example we don't have a `tag` key

Reply:
sorry for the late response. I was busy with other issues. Just updated the usage, thanks 👍